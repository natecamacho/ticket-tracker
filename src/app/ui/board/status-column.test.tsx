import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import StatusColumn from "./status-column";

describe("StatusColumn component", () => {
  it("renders initial StatusColumn component unchanged", () => {
    const { container } = render(<StatusColumn status="Test Status" />);

    expect(container).toMatchSnapshot();
  });
});
