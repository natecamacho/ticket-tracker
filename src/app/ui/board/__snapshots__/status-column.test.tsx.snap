// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusColumn component renders initial StatusColumn component unchanged 1`] = `
<div>
  <div
    class="flex w-80 h-12 bg-slate-200 mr-4 mt-4 items-center justify-center rounded-md"
  >
    Test Status
  </div>
</div>
`;
